server/webapp/WEB-INF/rails/webpack/views/components/materials/config_file_list_editor.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MithrilViewComponent} from "jsx/mithril-component";
+import _ from "lodash";
+import m from "mithril";
+import {ConfigFileList} from "models/materials/config_file_list";
+
+interface Attrs {
+  configFileList: ConfigFileList;
+}
+
+export class ConfigFileListEditor extends MithrilViewComponent<Attrs> {
```

Comment:
Not sure if this should be called "editor"? I don't think we are allowing the user any input here; mostly just informational display, but correct me if I'm wrong.

Reply:
@marques-work Well, I called it that because we eventually want to have it be a checkbox list for creating config repos. Editor might be the wrong suffix tho